app/pages/lab/workflows.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Link } from 'react-router';
+import DragReorderable from 'drag-reorderable';
+import ModalFormDialog from 'modal-form/dialog';
+import Paginator from '../../talk/lib/paginator';
+import WorkflowCreateForm from './workflow-create-form';
+import LoadingIndicator from '../../components/loading-indicator';
+
+const WorkflowsPage = (props) => {
+  const meta = props.workflows.length ? props.workflows[0].getMeta() : {};
+
+  const renderWorkflow = ((workflow) => {
+    return (
+      <li key={workflow.id}>
+        <Link key={workflow.id} to={props.labPath(`/workflow/${workflow.id}`)} className="nav-list-item" activeClassName="active">
+          {workflow.display_name}
+          {workflow.id === props.project.configuration.default_workflow && (
+            <span title="Default workflow">{' '}*{' '}</span>
+          )}
+        </Link>
+      </li>
+    );
+  });
+
+  return (
+    <div>
+      <div className="form-label">Workflows</div>
+
+      <DragReorderable tag="ul" className="nav-list" items={props.workflows} render={renderWorkflow} onChange={props.handleWorkflowReorder} />
```

Comment:
To make sure I understand this correctly. Aside from opening an issue on the UI for these pages, we are:

- Removing `DragReorderable` from this component

- Also removing Pagination from this page?